web-console/knikubevirt/Create-vm/step-5-review/wizard-review.md

```diff
@@ -0,0 +1,36 @@
+# Create Virtual Machine Wizard - Review Step
+
+The purpose of this doc is to update the current create VM designs to align with OpenShift and Patternfly 4.
```

Comment:
Let's remove this line, it makes more sense for the google doc probably not need for github.